avocado/utils/distro.py

```diff
@@ -17,10 +17,12 @@
 it's running under.
 """
 
+import logging
 import os
 import re
 import platform
 
+from . import project
```

Comment:
I see, so we need the value in utils as well. I'd recommend perhaps simply adding this value to `avocado.utils` directly. What do you think?